### PR TITLE
Documentation: fix broken relative link in quickstart.md

### PR DESCRIPTION
**Pull request by renaissanceGeek**

### CODE REVIEW

File: docs-src/content/getting-started/quickstart.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ weight: 2
 
 To evaluate Cobalt's speech recognition, you can access the [Cubic web demo](https://demo-cubic.cobaltspeech.com:8080) or can download a command-line interface built using our Go SDK to specify an audio file or list of files and get transcripts back from a running instance of cubicsvr. The [source code for cubic-cli](https://github.com/cobaltspeech/cubic-cli) is available to use as an example client.
 
-The command-line tool can call a local instance of Cubic (see [Installing the cubicsvr Image](cubic_docker.md)) or the demo server.
+The command-line tool can call a local instance of Cubic (see [Installing the Cubicsvr Image](/sdk-cubic/getting-started/cubic_docker)) or the demo server.
 
 ```bash
 # Local instance
```
Comment (catherinecobalt) [inline docs-src/content/getting-started/quickstart.md]: missing the "./" at the beginning of the call

File: docs-src/content/getting-started/quickstart.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ weight: 2
 
 To evaluate Cobalt's speech recognition, you can access the [Cubic web demo](https://demo-cubic.cobaltspeech.com:8080) or can download a command-line interface built using our Go SDK to specify an audio file or list of files and get transcripts back from a running instance of cubicsvr. The [source code for cubic-cli](https://github.com/cobaltspeech/cubic-cli) is available to use as an example client.
 
-The command-line tool can call a local instance of Cubic (see [Installing the cubicsvr Image](cubic_docker.md)) or the demo server.
+The command-line tool can call a local instance of Cubic (see [Installing the Cubicsvr Image](/sdk-cubic/getting-started/cubic_docker)) or the demo server.
```
Comment (catherinecobalt) [inline docs-src/content/getting-started/quickstart.md]: Should we link to terms and conditions of the demo server somewhere?

File: docs-src/content/getting-started/quickstart.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ weight: 2
...
```
Comment (catherinecobalt) [inline docs-src/content/getting-started/quickstart.md]: Which model does this use by default? Are there other things it's worth pointing out here, like how to change the model? What the output looks like?

File: docs-src/content/getting-started/quickstart.md (hunk 2 above)
Comment (renaissanceGeek) [reply docs-src/content/getting-started/quickstart.md]: Unfortunately, I don't think we have a legal t&c page for the demo server.  I remember that was on Scott's to-do list at one point, but I'm not sure what ended up happening.

File: docs-src/content/getting-started/quickstart.md (hunk 1 above)
Comment (renaissanceGeek) [reply docs-src/content/getting-started/quickstart.md]: fixed.  Of course, these two commands are equivalent, but it's good to be consistent so as not to cause confusion!

**Pull request merged**